pkg/antctl/raw/multicluster/command.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import "github.com/spf13/cobra"
+
+var GetCmd *cobra.Command
+
+func init() {
+	GetCmd = &cobra.Command{
+		Use:   "get",
+		Short: "List or query resources of multiple cluster",
```

Comment:
s/List or query resources of multiple cluster/Display one or many resources in a ClusterSet/